pcsx2/Config.h

```diff
@@ -293,7 +293,7 @@ struct Pcsx2Config
 		int FramesToDraw{ 2 }; // number of consecutive frames (fields) to render
 		int FramesToSkip{ 2 }; // number of consecutive frames (fields) to skip
 
-		double LimitScalar{ 1.0 };
+		double LimitPercent{ 1.0 };
```

Comment:
Nitpick: Not exactly a correct name when the value is 1.0 as this is the framelimiter on right?